tests/karma.local.config.js

```diff
@@ -0,0 +1,35 @@
+module.exports = (config) => {
+  config.set({
+    frameworks: ['jasmine', 'karma-typescript'],
+    basePath: '..',
+    files: [
+      'src/**/*.ts',
+      'tests/**/*.ts',
+      'dist/fp.min.js',
+    ],
+    preprocessors: {
+      '**/*.ts': 'karma-typescript',
+    },
+    reporters: ['spec', 'summary'],
+    browsers: ['ChromeHeadless', 'FirefoxHeadless'],
+
+    karmaTypescriptConfig: {
+      compilerOptions: {
+        ...require('../tsconfig.json').compilerOptions,
+        module: 'commonjs',
+        sourceMap: true,
+      },
+    },
+
+    specReporter: {
+      suppressErrorSummary: true,
+      suppressPassed: true,
+      suppressSkipped: true,
+      // todo: Suppress the summary completely when https://github.com/mlex/karma-spec-reporter/issues/83 is solved
```

Comment:
Looks like this repo is not maintained anymore :(

Reply:
Unfortunately. I don't want to spend much time and afford in this testing setup because it isn't battle-checked and there is a chance that it won't be able to run incognito testing. Maybe we will switch to another testing approach.